### DISCUSSION TITLE
Drag and drop and add event clicking directly to calendar

### DISCUSSION THREAD

**Discussion by scramatte** [wdelfuego/nova-calendar#31] [Ideas & Feature Requests]
Hi,

II would be very cool to have drag and drop to update events and add event clicking directly to calendar.

Regards

**Comment by wdelfuego** [comment]
Thanks for your input!

Event clicking works, but it opens a page within Nova (usually the detail or edit view of the related resource).

The problem to solve wrt in-calendar event editing is that a calendar can have events from different sources and with different related data models, so it's far from trivial to implement generic event editing elegantly. I agree that it'd be really nice to have the possibility though. I personally won't work on this anytime soon; other issues in this repo are more pressing for now (event filtering, week/day views).
